examples/codegen/README.md

```diff
@@ -0,0 +1,30 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+Example of how to run the pipeline:
+
+```python
+from examples.codegen.text_generation import TextGenerationPipeline
+
+codegen = TextGenerationPipeline(
+    model_path="/network/damian/static-codegen-350M-multi",
```

Comment:
do we have documentation on what's expected to be included in this directory?

Reply:
I'll make it a separate PR